packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,261 @@
+| [Incoming payment liquidity](#incoming-payment-liquidity-accounts) | The value received from a completed incoming payment | One per incoming payment |
+| [Outgoing payment liquidity](#outgoing-payment-liquidity-accounts) | The value that Rafiki will attempt to send in an outgoing payment | One per outgoing payment |
+| [Wallet address liquidity](#wallet-address-liquidity-accounts) | The value of an incoming payment received to a wallet address via either SPSP or Web Monetization | One per wallet address |
+
+</div>
+
+#### Asset liquidity accounts
+
+Asset liquidity ensures there is enough value available to handle a foreign exchange transaction between you and a peer. You have one asset liquidity account for each asset you transact in.
+
+An asset liquidity account represents the value that Rafiki has at its disposal for sending or forwarding ILP packets. The amount in an asset liquidity account increases when packets are received and decreases when packets are sent/forwarded. Any transaction that would result in a negative balance will fail. 
+
+:::note
+If you and your peer transact in the same asset (there's no currency conversion) and you both provide your customers only with wallet addresses denominated in that asset, then there will be no movement info/from the corresponding asset liquidity account. In this scenario, the transaction occurs over Interledger's Simple Payment Setup Protocol (SPSP).
+
+For example, you and your peer transact in USD and only provide your customers with USD wallet addresses. 
+:::
+
+You should define and adjust asset liquidity based on your liquidity risk profile. You can deposit or withdraw asset liquidity as needed through [Rafiki Admin](/admin/admin-user-guide#edit-asset) or by using the [Backend Admin API](/admin/manage-liquidity#asset-liquidity).
+
+<details>
+<summary>Asset liquidity example - cross-currency transactions</summary>
+Your Rafiki instance is configured for two assets: EUR and USD. 
 
-A settlement account holds a non-positive balance, with Rafiki ensuring that total credits do not exceed total debits. A settlement account represents the total funds you as the ASE have deposited into Rafiki.
+* Rafiki holds an asset liquidity account for both EUR and USD. 
+* You’ve set the asset scale of both currencies to 0.
+* Your starting EUR liquidity is 10 and your USD liquidity is 50.
 
-Before you can peer with another ASE, you must both agree on the asset you will use for settlement. The Interledger packets exchanged between you and your peer will be denominated in the agreed-upon asset.
+**Cross-currency transaction #1:**
 
-There is one settlement account for each asset.
+<Steps>
+1. Rafiki receives packets worth 10 EUR. Your EUR liquidity increases to 20 (10 + 10).
+2. The current EUR-to-USD exchange rate is applied, with 10 EUR equating to 12 USD.
+3. Rafiki sends packets worth 12 USD. Your USD liquidity decreases to 38 (50 - 12).
```

Comment:
Rafiki moves funds to an incoming payment for 12 USD from the USD liquidity account.

Reply:
What you wrote should be step 3, and the current step 3 becomes step 4, correct?

Reply:
Given this is from the perspective of the receiver, nothing happens when sending the fulfillment packets back. So after the funds are moved to the incoming payment from the liquidity account, there is nothing left to be done on the receiving side from the accounting perspective

Reply:
I've updated the steps. Please take a look to make sure I understand when I push the next set of changes.